clients/config_client/config_client.go

```diff
@@ -138,7 +138,8 @@ func NewConfigClient(nc nacos_client.INacosClient) (*ConfigClient, error) {
 
 	config.cacheMap = cache.NewConcurrentMap()
 	// maximum buffered queue to prevent chan deadlocks during frequent configuration file updates
-	config.listenExecute = make(chan struct{}, math.MaxInt64)
+	// use Math.MaxInt to avoid overflows on ARM 32-bits
+	config.listenExecute = make(chan struct{}, math.MaxInt)
```

Comment:
Sorry, maxInt is only supported in golang 1.17, so use maxInt32 instead

Reply:
OK, I changed it to MaxInt and made corresponding tests and it works normally